posnum.c

```diff
@@ -0,0 +1,111 @@
+/* Jinoo Hong */
+/* Write a program `posnums.c` that reads 2 positive integers from the user, and prints the average of the two (this should be printed as a floating point number.), whether each number is a prime number or not, and whether the two numbers are relatively prime or not (the only positive integer that divides both of them evenly is 1). */
+
+#include <stdio.h>
```

Comment:
To avoid this in the future, make sure to test your code as much as possible and thoroughly check the logic.